lib/convert_query_type.js

```diff
@@ -0,0 +1,108 @@
+var ObjectId = require('mongodb').ObjectId;
+
+/**
+ * Creates an object containing the expression key and value queried.
+ *
+ * @param {String} expression - This is the operator used in query.
+ * @param {String} field - This is the column being quried.
+ * @param {ObjectId|Number|Boolean} value - This is the value being quried.
+ *
+ * @returns {Object} expressionQuery - contains the expressions key and query value.
+ */
+function expressionQuery(value, field, expression) {
```

Comment:
We can't call it query because it's being invoked inside the "Any" function where query has been defined as on object. So it will throw an error saying it's not a function. I'll try come up with a better name